tests/test_scaler.py

```diff
@@ -86,9 +86,11 @@ def test_scaler_functionality(self):
         str(sca)
 
         sca.stage()
-        sca.configure()
+        old, new = sca.configure({})
```

Comment:
It may also be a consequence of using a 'fake' scaler in the test, though I haven't really looked into it yet.